components/TextField/index.jsx

```diff
@@ -20,6 +22,8 @@ export default class TextField extends React.Component {
   static propTypes = {
     autoComplete: PropTypes.string,
     autoFocus: PropTypes.bool,
+    badwordsCallback: PropTypes.func,
+    badwordsList: PropTypes.object,
```

Comment:
I'm not sure, if badwords is the best name for this. Can we find something more generic, like highlight?

Reply:
I thought about this too - the problem I see is with the color. The kununu design requires red text with light red background. This is very specific to errors. If we actually provide just highlighting then we would need to allow the user to change the colors (which would also allow them to change other things and potentially break the styles) or we specify certain types like basic, error, alert etc.. but since we don't have a use case I am not sure about this either.

Reply:
Ok. However regarding the color, I see a different problem as well. When someone uses this component and overrides our styling, he will need to override the styling for the pseudo text box as well. Maybe this is something, that should be mentioned in a comment or somewhere.

Reply:
many plus here!

Reply:
badwords is not good name for this feature
maybe highlightErrors highlightWarnings is better

Reply:
@codingcircus the point I was trying to make is that I don't think we should add functionality that we do not need currently. Right now we only highlight badwords with a red background. So let's implement this.

I am open to other suggestions for naming though. However, @dennisat we are not highlighting errors nor warnings here so I don't think those names make sense.